### PR TITLE
url parsing fix to account for different urls in versions of python plus import fix

### PR DESCRIPTION
**Pull request by williexu**
-added same fix from sdk to python devtools
-subs `//` for `/` in middle of url

### CODE REVIEW

**Review by lmazuel — APPROVED** [review]
LGTM!

File: src/azure_devtools/scenario_tests/base.py [hunk 1]
```diff
@@ -142,6 +143,11 @@ def tearDown(self):
         os.environ = self.original_env
 
     def _process_request_recording(self, request):
+        # URL parsing fix to account for '//' vs '/' in different versions of python
+        # - Python 2.7 for 2.7/3.3/3.4 (...Microsoft.Compute//availabilitySets...)
+        # - Python 3.5 (...Microsoft.Compute/availabilitySets...)
+        request.uri = re.sub('(?<!:)//', '/', request.uri)
+
```
Comment (troydai) [inline src/azure_devtools/scenario_tests/base.py]: Please put this in a separate recording processor and add it to the other recording processors. The actual logic of altering the recording should not be placed in the framework.

**Review by troydai — CHANGES_REQUESTED** [review]

File: src/azure_devtools/scenario_tests/recording_processors.py [hunk 2]
```diff
@@ -175,3 +175,11 @@ def process_response(self, response):
             self.replace_header(response, 'azure-asyncoperation', old, new)
 
         return response
+
+
+class SlashReplacer(RecordingProcessor):
```
Comment (troydai) [inline src/azure_devtools/scenario_tests/recording_processors.py]: A better name is like `RequestUrlNormalizer`

**Review by troydai — APPROVED** [review]
Rename the class. Otherwise looks good.

**Pull request merged**